RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress.swift

```diff
@@ -352,31 +350,3 @@ extension AccountRecoveryScanInProgress {
 		}
 	}
 }
-
-extension DerivationPath {
```

Comment:
moved to https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage1/Stage1MigrateToSargon%2BAccount.swift#L17